### DISCUSSION TITLE
Problem with Alias

### DISCUSSION THREAD

**Discussion by Kyou04** [microsoft-search/pnp-modern-search#3451] [Q&A]
**Version used**
Ex: 4.8

**Describe the bug**
When I search a result with alias , like "Alias:xxxxx" I have no result. But when i use the property Name , I have 2 results

How is possible to have different result only in using Alias ? How can I fix it ? 

Result with Property Name
![image](https://github.com/microsoft-search/pnp-modern-search/assets/54059745/4294315d-cb8c-40e3-9ee3-84729530d719)

Result with Alias 
![image](https://github.com/microsoft-search/pnp-modern-search/assets/54059745/761264c7-1df7-4a86-999b-c7e63ea75218)

Here the configuration : 
![image](https://github.com/microsoft-search/pnp-modern-search/assets/54059745/657258a4-66b2-4e52-8c32-10a1eda7b7e5)

And the result with standard Sharepoint Search 
![image](https://github.com/microsoft-search/pnp-modern-search/assets/54059745/c2cbc126-ea41-43b2-9fe2-5831e19ba8fd)

Concretly I need the result when I use Alias on Pnp - SearchBox return the same result then when I use the Property Name. 

**Desktop (please complete the following information):**
 Google Chrome.

**Comment by wobba** [comment]
Which version are you using?

**Reply by Kyou04** [reply]
4.8.0

![image](https://github.com/microsoft-search/pnp-modern-search/assets/54059745/1e480baa-65fe-40b2-b864-afad8adc719b)

**Reply by wobba** [reply]
I cannot repro as alias queries seem to work fine when I test. Testing in v4.10 and via the SP Editor browser plugin. Also, using punctuation and wildcards may break tokenization and search scenarios. I assume you have the number as text due to the * at the end. I would add () around the text value to ensure they are all handled towards the property like `Numero:(123.123*)`.

Also, did you add the alias on the site schema or tenant schema, and if the site schema, is the web part hosted on the same site?

**Reply by Kyou04** [reply]
Hello, I try with () but the problem is the same. 

Can I send you something that could help you understand the problem?
Anything that could explain why there is a difference between the pnp-modern-search results and the standard Sharepoint search?

**Reply by wobba** [reply]
@Kyou04 open the network tab in dev tools in your browser, make a copy of the JSON payload on the "postquery", assuming you are using the SharePoint provider. Or are you using the Microsoft Search provider? Then filter on "query" to find the request. Looking at the request payload will help figure out the issue.
